lib/specs/v2.js

```diff
@@ -498,6 +498,96 @@ let rules = {
         className: '.kg-gallery-image',
         css: true
     },
+    'GS050-CSS-KGBM': {
+        level: 'error',
+        rule: 'The <code>.kg-bookmark-card</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-bookmark-card</code> CSS class is required otherwise the bookmark card will appear unstyled.
+        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/#bookmark-card" target=_blank>here</a>.`,
+        regex: /\.kg-bookmark-card/g,
+        className: '.kg-bookmark-card',
+        css: true
+    },
+    'GS050-CSS-KGBMCO': {
+        level: 'error',
+        rule: 'The <code>.kg-bookmark-container</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-bookmark-card</code> CSS class is required otherwise the bookmark card will appear unstyled.
+        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/#bookmark-card" target=_blank>here</a>.`,
+        regex: /\.kg-bookmark-container/g,
+        className: '.kg-bookmark-container',
+        css: true
+    },
+    'GS050-CSS-KGBMCON': {
+        level: 'error',
+        rule: 'The <code>.kg-bookmark-content</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-bookmark-content</code> CSS class is required otherwise the bookmark card will appear unstyled.
+        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/#bookmark-card" target=_blank>here</a>.`,
+        regex: /\.kg-bookmark-content/g,
+        className: '.kg-bookmark-content',
+        css: true
+    },
+    'GS050-CSS-KGBMTI': {
+        level: 'error',
+        rule: 'The <code>.kg-bookmark-title</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-bookmark-title</code> CSS class is required otherwise the bookmark card will appear unstyled.
```

Comment:
```suggestion
        details: oneLineTrim`The <code>.kg-bookmark-title</code> CSS class is required otherwise the bookmark card title will appear unstyled.
```